neutron/include/scipp/neutron/diffraction/convert_with_calibration.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+/// @file
+/// @author Mads Bertelsen
+#ifndef SCIPP_NEUTRON_CONVERT_WITH_CALIBRATION_H
+#define SCIPP_NEUTRON_CONVERT_WITH_CALIBRATION_H
+
+#include <vector>
```

Comment:
Not required?

Reply:
I assume you mean the <vector>? If so that was removed. The header protection is still required as I understand it.

Reply:
I mean `#include <vector>`

Reply:
Great. It seems the github tool for showing lines in reviews take a few extra lines before for no reason.